.github/workflows/teardown_datamanager.yaml

```
@@ -0,0 +1,39 @@
+jobs:
+  teardown:
+    name: Teardown datamanager service
+    runs-on: ubuntu-latest
+    environment: pulumi
+    steps:
+      - name: Check if within market hours
+        id: check-hours
+        run: |
+          current_time=$(TZ='America/New_York' date '+%H%M')
+          current_day=$(TZ='America/New_York' date '+%u')  # 1=Monday, 7=Sunday
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)